### PR TITLE
[WIP] Full view

### PR DESCRIPTION
**Pull request by arifrag**

### CODE REVIEW

**Comment by arifrag** [comment]
add fiture show all and compact in purchasing

File: packages/point/point-purchasing/src/Http/Controllers/Inventory/InvoiceController.php [hunk 1]
```diff
@@ -33,11 +34,22 @@ public function index()
     {
         $view = view('point-purchasing::app.purchasing.point.inventory.invoice.index');
         $list_invoice = Invoice::joinFormulir()->joinSupplier()->notArchived()->selectOriginal();
+        // dd($list_invoice);
```
Comment (martiendt) [inline packages/point/point-purchasing/src/Http/Controllers/Inventory/InvoiceController.php]: please remove this comment

File: packages/point/point-purchasing/src/Http/Controllers/Inventory/InvoiceController.php [hunk 2]
```diff
@@ -33,11 +34,22 @@ public function index()
...
         $list_invoice = InvoiceHelper::searchList($list_invoice, \Input::get('order_by'), \Input::get('order_type'), \Input::get('status'), \Input::get('date_from'), \Input::get('date_to'), \Input::get('search'));
         $view->list_invoice = $list_invoice->paginate(100);
+     
+        $data_id = [];
```
Comment (martiendt) [inline packages/point/point-purchasing/src/Http/Controllers/Inventory/InvoiceController.php]: use semantic name like $invoice_id

File: packages/point/point-purchasing/src/views/app/purchasing/point/inventory/invoice/index.blade.php [hunk 3]
```diff
@@ -70,7 +73,8 @@
                         </thead>
                         <tbody>
                         @foreach($list_invoice as $invoice)
-                            <tr id="list-{{$invoice->formulir_id}}">
+                            <?php array_push($data_id,$invoice->id); ?>
+                            <tr class="rowDetail" id="row_detail_{{$invoice->id}}">
```
Comment (martiendt) [inline packages/point/point-purchasing/src/views/app/purchasing/point/inventory/invoice/index.blade.php]: don't use camel case for class name instead use class="long-name"

File: packages/point/point-purchasing/src/views/app/purchasing/point/inventory/purchase-requisition/index.blade.php [hunk 4]
```diff
@@ -104,6 +109,43 @@
...
+    var check_show = $('#check_show').val();
+    var data_id = $('#data_id').val();
+    $('#check_show').val(1);
+    if(check_show == 0){
+        var temp = data_id.split('#');
+        for (var x = temp.length - 1; x >= 0; x--) {
+            var str_url = "{{ url('purchasing/point/purchase-requisition/detail') }}/"+temp[x];
+            $.ajax({ url:str_url, success: function(data) {
+                for (var i = 0; i < data.length; i++) {
+                    var html_detail = ' <tr class="data_detail">'
```
Comment (martiendt) [inline packages/point/point-purchasing/src/views/app/purchasing/point/inventory/purchase-requisition/index.blade.php]: what is html_detail ?

what is data_detail ?

**Pull request merged**